IA/IA_upload.py

```diff
@@ -33,16 +33,18 @@ def mp_from_ids(mp_id: str, mp_keyname: str, bucket: str) -> MultiPartUpload:
 
     tasks = []
 
-    for root, dirs, files in os.walk(parent):
+    directory_path = os.path.join(HERE, parent)
+    for root, dirs, files in os.walk(directory_path):
         for file in files:
             path = os.path.join(root, file)
             with open(path, 'rb') as fp:
                 data = fp.read()
                 size = len(data)
+                url_path = path.split(parent+'/')[1]
```

Comment:
shouldn't flake catch this because of https://lintlyci.github.io/Flake8Rules/rules/E225.html ?

Reply:
The rules kind of change when theres an operator in arguments. Flake8 doesn't seem to enforce anything. PEP8 seems to say use best judgement when putting whitespace around operators in arguments. I'll stick whitespaces around that though!

Reply:
> Flake8 doesn't seem to enforce anything. 

Mine is rejecting things of travis and the CLI is working.